mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.deleteOrder = async (req, res) => {
+  try {
+    const order = await Order.findByIdAndRemove(req.params.orderId);
+    if (!order) {
+      return res.status(404).json({ message: "Order does not exist" });
+    }
+    res.status(200).json({ message: "Order deleted successfully" });
+  } catch (err) {
+    res.status(500).json({ error: errorHandler(err) });
```

Comment:
In the error handling, it would be better to include a more descriptive error message instead of just sending the error object. This will provide more context to the user about what went wrong. For example, you could include a message like "Internal Server Error" or "Failed to delete order". 

Also, consider logging the error on the server-side for debugging purposes. This will help in identifying and fixing issues more efficiently.